src/components/User/User.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const UserContainer = styled.div`
```

Comment:
This can be brought to a separate file to extract styles from component declaration

Reply:
Done, exported to User.styled.js